### PR TITLE
Modern issue/feature template

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### What does this implement/fix? Explain your changes.
Issue/bug report templates were in markdown format. This PR will update that template's format to `.yml` format.
Also, a separate feature request template is added.

### CODE REVIEW

**Review by AHReccese — APPROVED** [review]
Interesting update, thanks.

**Review by AHReccese — APPROVED** [review]

File: otherfiles/RELEASE.md [hunk 1]
```diff
@@ -22,17 +24,17 @@
...
 	2. Tag: `vx.x` (Example: `v0.1`)
 	3. Title: `Version x.x` (Example: `Version 0.1`)
 	4. Copy changelogs
 	5. Tag all related issues
-7. Bump!!
-8. Close this version issues
-9. Close milestone
\ No newline at end of file
+8. Bump!!
+9.  Close this version issues
```
Comment (sepandhaghighi) [inline otherfiles/RELEASE.md:39]: Extra space?

File: otherfiles/RELEASE.md [hunk 2]
```diff
@@ -13,7 +13,9 @@
 	1. Add a new header under `Unreleased` section (Example: `## [0.1] - 2022-08-17`)
 	2. Add a new compare link to the end of the file (Example: `[0.2]: https://github.com/openscilab/pymilo/compare/v0.1...v0.2`)
 	3. Update `dev` compare link (Example: `[Unreleased]: https://github.com/openscilab/pymilo/compare/v0.2...dev`)
-4. Create a PR from `release` to `dev`
+4. Update `bug_report.yml`
```
Comment (sepandhaghighi) [inline otherfiles/RELEASE.md:16]: I suggest using the full path:
`.github/ISSUE_TEMPLATE/bug_report.yml`

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@sadrasabouri 🔥

**Review by sepandhaghighi — APPROVED** [review]
🚀

**Pull request merged**